tests/packages/Test.hs

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env stack
+{- stack
+     --resolver lts-6.6
+     --install-ghc
+     runghc
+     --package turtle
+     --package text
+     --package aeson
+     --package bytestring
+     --package directory
+-}
+
+{-#LANGUAGE OverloadedStrings#-}
+
+module Test where
+
+import GHC.IO.Exception (ExitCode(..))
+import System.Exit (die)
+import Data.Monoid ((<>))
+import Data.Text (unpack, Text)
+import Control.Applicative (empty)
+import Data.Aeson
+import Turtle.Prelude (shell)
+import qualified Data.ByteString.Lazy as BS
+import System.Directory (getHomeDirectory)
+
+data Packages = Packages {
+      patched :: [Text],
+      vanilla :: [Text]
+    } deriving (Show, Eq, Ord)
+
+instance FromJSON Packages where
+    parseJSON (Object v) = Packages <$> v.: "patched" <*> v.: "vanilla"
+    parseJSON _ = empty
+
+parsePackagesFile :: FilePath -> IO (Maybe Packages)
+parsePackagesFile fname = do
+  contents <- BS.readFile fname
+  let packages = decode contents
+  return packages
+
+packagesFilePath :: IO FilePath
+packagesFilePath = (<> "/.cabalvm/packages.json") <$> getHomeDirectory
```

Comment:
Note that I'm using `</>` from the package `filepath`. Let me know if you had something different in mind.